programs/gzip.c

```diff
@@ -43,11 +43,12 @@ struct options {
 	bool decompress;
 	bool force;
 	bool keep;
+	bool test;
 	int compression_level;
 	const tchar *suffix;
 };
 
-static const tchar *const optstring = T("1::2::3::4::5::6::7::8::9::cdfhknS:V");
+static const tchar *const optstring = T("1::2::3::4::5::6::7::8::9::cdfhkntS:V");
```

Comment:
Please keep the options in alphabetical order.

Reply:
Sure